std/math/fma.zig

```diff
@@ -13,6 +14,7 @@ pub fn fma(comptime T: type, x: T, y: T, z: T) T {
     return switch (T) {
         f32 => fma32(x, y, z),
         f64 => fma64(x, y, z),
+        f128 => fma128(x, y, z),
```

Comment:
Where is ```fma128``` implementation?

Reply:
whoops. I was working on it, and that accidentally god added.